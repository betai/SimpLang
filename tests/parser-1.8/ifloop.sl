    if 1 then 1 else loop x=1 in recur (x) end end
