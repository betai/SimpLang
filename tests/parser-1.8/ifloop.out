if
  1
  1
  loop
      x
        1
    recur
      x
